core/src/com/unciv/ui/cityscreen/CityConstructionsTable.kt

```diff
@@ -444,6 +444,8 @@ class CityConstructionsTable(private val cityScreen: CityScreen) {
             is BaseUnit -> UncivSound.Promote
             PerpetualConstruction.gold -> UncivSound.Coin
             PerpetualConstruction.science -> UncivSound.Paper
+            PerpetualConstruction.culture -> UncivSound.Chimes
+            PerpetualConstruction.faith -> UncivSound.Choir
```

Comment:
I think this is kind of long and reserved for finishing wonders, settling cities etc

Reply:
I changed the `UncivSound.Chimes` to `UncivSound.Policy` which is a shorter sound but is still related to culture.